versioned_docs/version-8.4/components/modeler/desktop-modeler/troubleshooting.md

```diff
@@ -56,6 +56,12 @@ You try to connect (i.e., to deploy) to a remote Zeebe instance, and Desktop Mod
 
 To resolve this issue, check if you can connect to Zeebe through another client, i.e., [`zbctl`](../../../apis-tools/cli-client/index.md). If that works, [further debug your Zeebe connection](#debug-zeebe-connection-issues). If that does not work, resolve the [general connection issue](#resolve-a-general-zeebe-connection-issue) first.
 
+## I cannot connect to local Zeebe
+
+You try to connect (i.e., to deploy) to a local Zeebe instance, and Desktop Modeler tells you it "cannot find a running Zeebe."
+
+Ensure that your local Zeebe instance is running. If you don't have one installed, consider [c8run](/self-managed/setup/deploy/local/c8run.md), a lightweight all-in-one distribution.
```

Comment:
```suggestion
Ensure that your local Zeebe instance is running.
```

We may need to remove the link given the CI failure: https://github.com/camunda/camunda-docs/actions/runs/16044577648/job/45272757195#step:7:13